docs/docs/spark-procedures.md

```diff
@@ -1055,4 +1056,4 @@ metadata files and data files to the target location.
 Lastly, the [register_table](#register_table) procedure can be used to register the copied table in the target location with a catalog.
 
 !!! warning
-    Iceberg tables with partition statistics files are not currently supported for path rewrite.
\ No newline at end of file
+    Iceberg tables with partition statistics files are not currently supported for path rewrite.
```

Comment:
Unnecessary newline change?

Reply:
Yeah seems like my IDE is adding these

Reply:
Retried the same approach through online git file edit. from UI as well.  It seems like adding the new option is automatically showing line 1059 as a change.

Reply:
There is an extra " " at the end of this line that you are removing.

"for path rewrite " vs "for path rewrite"

Reply:
Double never mind, there it is
![image](https://github.com/user-attachments/assets/3525bf17-5d5b-45d4-a96e-24557b369990)

Reply:
@RussellSpitzer  I tried editing the line from Github repo , sublime etc but for some reason my commits wouldnt work :|